### PR TITLE
feat(storage): adds filter for own storage listing

### PR DESCRIPTION
**Pull request by jurajpiar**
Adds a filter option for account (to be filtered out).

Resolves https://rsklabs.atlassian.net/browse/RMKT-395

Works with cache PR https://github.com/rsksmart/rif-marketplace-cache/pull/399

### CODE REVIEW

File: src/context/Services/storage/offers/Context.tsx [hunk 1]
```diff
@@ -113,13 +120,21 @@ export const Provider = ({ children }) => {
         //   payload: { refresh: true },
         // } as any)
         // setTimeout(() =>
+
+        dispatch({
+          type: 'FILTER',
+          payload: {
+            account,
+          },
+        })
```
Comment (farinalvaro) [inline src/context/Services/storage/offers/Context.tsx:129]: I might be missing something but why do we filter by the current account?
As we have the provider of every offer, could we just compare it to the current account?

Comment (farinalvaro) [reply src/context/Services/storage/offers/Context.tsx:129]: Also, I think we need to display that offer but just prevent the user from buying it

Comment (jurajpiar) [reply src/context/Services/storage/offers/Context.tsx:129]: > As we have the provider of every offer, could we just compare it to the current account?

Filtering via SQL is less expensive.

> Also, I think we need to display that offer but just prevent the user from buying it

Fair enough. I guess that's the way it is done in rns as well, after all. In that case, I'd do it locally of course.

**Comment by jurajpiar** [comment]
Closing as there was a misunderstanding of requirements on my part and it's going to be simpler to create a new branch than modify this one.

**Pull request closed without merging**